schemas/constructs/v1beta1/academy/merged-openapi.yml

```diff
@@ -0,0 +1,138 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Academy",
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/api/academy/learningPaths": {
```

Comment:
Use kebab casing, please.

Reply:
we can still go with kebab case starting with this